src/motor_control.c

```diff
@@ -87,7 +87,17 @@ void motor_control_apply(MotorControl *mc, float abs_erpm, RunState state, float
         VESC_IF->mc_set_current(mc->requested_current);
     } else {
         // Brake logic
-        if (abs_erpm > ERPM_MOVING_THRESHOLD) {
+        if (mc->parking_brake_mode == PARKING_BRAKE_IDLE) {
+            // only evaluate parking brake each cycle if pbmode is IDLE
+            if ((state != STATE_RUNNING) &&
+                ((md->abs_erpm_smooth < 50) || (md->last_erpm_sign != md->erpm_sign))) {
+                mc->parking_brake_active = true;
+            } else {
+                mc->parking_brake_active = false;
+            }
+        }
```

Comment:
I like this change to the logic (where the ALWAYS/NEVER cases are handled in configure), it's easier to understand this way :+1:.

But is there a reason you put this into the "brake logic" branch, meaning it's not evaluated when there's regular current requested? I'm not sure of the implications right now...